pkg/web/static/styles/style.css

```diff
@@ -25,15 +26,16 @@ body main {
   @apply mb-1 block font-bold;
 }
 
-.envelope-header {
-  @apply text-balance text-sm text-left sm:text-base;
-}
-
-.envelope-data {
-  @apply text-sm sm:text-base p-2 md:px-6 md:py-4 max-w-44 break-words;
+#transaction-rejection {
```

Comment:
This replaces the select arrow with a hero icon and then styles the element so that there's more space to the right of the element. daisyui added a thick border I had trouble getting rid of and Slim Select wouldn't display the options, so this felt like the best way to resolve this for now.